src/components/AttendantForm.jsx

```diff
@@ -0,0 +1,320 @@
+import React, { useState } from "react";
+import Attendant from "./Attendant";
+import ErrorMessages from "../constants/errorMessages";
+
+const AttendantForm = ({
+  isAttendantsLoading,
+  isJobTitlesLoading,
+  submitAttendant,
+  attendants,
+  jobTitles,
+  attendantsApiError,
+  jobTitlesApiError,
+}) => {
+  const [name, setName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [jobTitle, setJobTitle] = useState("");
+  const [age, setAge] = useState("");
+  const [nameError, setNameError] = useState("");
+  const [lastNameError, setLastNameError] = useState("");
+  const [jobTitleError, setJobTitleError] = useState("");
+  const [ageError, setAgeError] = useState("");
+
+  async function handleSubmit(e) {
+    e.preventDefault();
+    await submitAttendant({ name, lastName, jobTitle, age });
+    clearForm();
+  }
+
+  const clearForm = () => {
+    [setName, setLastName, setJobTitle, setAge].forEach((setter) => setter(""));
+  };
+
+  const renderAttendants = () =>
+    attendants.map((attendant, index) => (
+      <Attendant
+        key={`attendant-${index}`}
+        attendant={attendant}
+      />
+    ));
+
+  const renderJobTitles = () =>
+    jobTitles.map((jobTitle, i) => (
+      <option
+        key={`${jobTitle}${i}`}
+        value={jobTitle}
+      >
+        {jobTitle}
+      </option>
+    ));
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    switch (name) {
+      case "name":
+        setName(value);
+        validateNameInput(value);
+        break;
+      case "lastName":
+        setLastName(value);
+        validateLastNameInput(value);
+        break;
+      case "jobTitle":
+        setJobTitle(value);
+        validateJobTitleSelection(value);
+        break;
+      case "age":
+        setAge(value);
+        validateAgeInput(value);
+        break;
+      default:
+        break;
+    }
+  };
+
+  const handleBlur = (e) => {
```

Comment:
This and `handleChange` functions seem to do almost the same. I suggest having a separate function that would handle the validation for both `handleBlur` and `handleChange`.